content/ADR-231-voting-power-usage-in-dapps

```diff
@@ -0,0 +1,37 @@
+---
+layout: adr
+adr: 231
+title: Voting Power usage for High Rated Places and Marketplace Saved Items
+date: 2023-05-09
+status: Draft
+type: Standards Track
+spdx-license: CC0-1.0
+authors:
+  - kevinszuchet
+---
+
+## Abstract
+
+This document outlines the guidelines implemented in the dApps to use the Voting Power (VP) against abuse. Given that some dApps such as Places and the Marketplace have use cases where users can vote, rank, or save resources for later use, the Voting Power could help determine the number of reliable users interacting with them.
+
+## Context, Reach & Prioritization
+
```

Comment:
Can we add context about why we use the VP and how it is being used in our dapps